app/posts/manage-breast-screening/2025/05/2025-05-06-image-reading-first-prototype.md

```diff
@@ -0,0 +1,112 @@
+
+![A screenshot of our image reading dashboard](technical-recall.png)
+
+### Recall for assessment
+
+In NBSS, there are two options that lead to a participant being recalled for further assessment: 'clinical' and 'abnormal'.
+
+The 'clinical' option is used when images appear normal, but the participant reported symptoms that require clinical investigation. The 'abnormal' option is used when the radiologist sees something concerning in the images.
+
+Since both options lead to the same next stage (recall for assessment), we're considering combining them into a single initial option in our design.
+
+We let the reader give an opinion per breast, and if there were symptoms reported, one option is to recall for clinical reasons.
+
+![Recall for assessment screen with radio choice per breast to give an opinion](recall-for-assessment.png)
+
+In NBSS, radiologists give their opinion per breast and note the types of issues they see on the breast as a whole. We're trialling a different approach: recording observed issues as specific annotations, allowing multiple annotations per breast.
+
+This would provide greater precision – for example, a radiologist could indicate a well-defined mass in the lower inner quadrant and a lymph node abnormality in the upper outer quadrant. Currently, they can only record that the breast contains both abnormalities without specifying their locations.
+
+The trade-off is that recording this more structured data may take radiologists slightly longer, which we'll need to evaluate in testing.
+
+![Recall for assessment screen with radio choice per breast to give an opinion](add-breast-annotation.png)
+
+### Reviewing opinions
+
+When radiologists complete reading all cases in a clinic, they return to the clinic list page where they can review all their opinions and any cases they skipped. From there, they can navigate back to the main image reading section or revisit specific cases if needed.
+
+![The clinic list page showing all cases with their read opinion](clinic-list-complete.png)
+
+## How it tested
+
+Our testing of this prototype was largely positive, with a few issues identified.
+
+Users were able to easily navigate to a clinic and start reading. The options were familiar to them or they easily found equivalents. The general workflow of reading an individual case is similar to the current process – though a big change is in not having medical information to hand in paper form and instead needing to look for it.
+
+The largest issue observed was in adding annotations. Partly this is because the design is unfinished – the intention is that users can loop through the annotations and add multiple observations. The prototype has a button to ‘select location on image’ - but this was unfunctional in testing. Without selecting a specific area on the breast, users felt that the whole page felt burdensome – why not just collect this info on the main recall page?
```

Comment:
```suggestion
The largest issue observed was in adding annotations. Partly this is because the design is unfinished – the intention is that users can loop through the annotations and add multiple observations. The prototype has a button to ‘select location on image’ - but this was not functional in testing. Without selecting a specific area on the breast, users felt that the whole page felt burdensome – why not just collect this info on the main recall page?
```